airflow/cli/cli_config.py

```diff
@@ -277,7 +277,12 @@ def string_lower_type(val):
 ARG_NO_BACKFILL = Arg(
     ("--no-backfill",), help="filter all the backfill dagruns given the dag id", action="store_true"
 )
-ARG_STATE = Arg(("--state",), help="Only list the dag runs corresponding to the state")
+ARG_STATE = Arg(
+    ("--state",),
+    help="Only list the dag runs corresponding to the state",
+    metavar="(running, queued, success, failed)",
+    choices=("running", "queued", "success", "failed"),
+)
```

Comment:
Nit: I always have a preference to extract the possible values dynamically when they are based on another class/enum. But I have no objection to defining them this way.
```suggestion
dagrun_states = tuple(state.value for state in DagRunState)
ARG_STATE = Arg(
    ("--state",),
    help="Only list the dag runs corresponding to the state",
    metavar=str(dagrun_states),
    choices=dagrun_states,
)
```

Reply:
Instead of `str(dagrun_states)` it’s probably prettier to have `", ".join(dagrun_states)`?

Reply:
@uranusjr should it be enclosed in parentheses to align with the other arguments?
```python
metavar=f'({", ".join(dagrun_states)})',
```